src/lib_radare2.pri

```diff
@@ -0,0 +1,73 @@
+win32 {
+    DEFINES += _CRT_NONSTDC_NO_DEPRECATE
+    DEFINES += _CRT_SECURE_NO_WARNINGS
+    INCLUDEPATH += "$$PWD/../iaito_win32/include"
+    INCLUDEPATH += "$$PWD/../iaito_win32/radare2/include/libr"
+    !contains(QMAKE_HOST.arch, x86_64) {
+        LIBS += -L"$$PWD/../iaito_win32/radare2/lib32"
+    } else {
+        LIBS += -L"$$PWD/../iaito_win32/radare2/lib64"
+    }
+} else {
+    # pseudo auto detection of libr paths, in the following order:
```

Comment:
As @trufae commented here:
https://github.com/hteso/iaito/pull/5#issuecomment-290433796
It is actually possible to get the exact location of libr, without adding a dependency, with something like:
```
r2 -H | grep INCDIR | sed 's/[^=]*=//'
```